packages/spotlight/Spottable/Spottable.js

```diff
@@ -270,13 +271,15 @@ const Spottable = hoc(defaultConfig, (config, Wrapped) => {
 				// control
 				(
 					which === value &&
-					(type !== 'keypress' || !isKeyboardAccessible(currentTarget))
+					(type !== 'keypress' || !keyboardAccessible)
 				)
 			));
 
 			if (getDirection(keyCode)) {
 				preventDefault(ev);
 				stop(ev);
+			} else if (keyboardAccessible) {
```

Comment:
I think we should further limit it to matching selection keys. Right now, it will call `preventDefault` for other key presses which I don't think is desired. Do you agree?

Reply:
Ah, good point. I mistakenly thought there was a guard for that in place already. I've updated this line to include it. Good 👀